capa/main.py

```diff
@@ -589,6 +593,30 @@ def get_rules(rule_paths: List[str], disable_progress=False) -> List[Rule]:
                     rule_path = os.path.join(root, file)
                     rule_file_paths.append(rule_path)
 
+    return rule_file_paths
+
+
+def get_rules(rule_paths: List[str], disable_progress=False) -> RuleSet:
```

Comment:
breaking change: this routine now returns the ruleset, not list of rule instances